frameworks/helloworld/src/main/dist/examples/uri.yml

```diff
@@ -35,7 +36,7 @@ pods:
           type: ROOT
           size: {{WORLD_DISK}}
     uris:
-      - "https://github.com/mesosphere/dcos-commons/blob/master/frameworks/helloworld/README.md"
+      - https://github.com/mesosphere/dcos-commons/blob/master/frameworks/helloworld/README.md
```

Comment:
Out-of-scope: Would it make sense to also strip these kinds of characters from URIs when constructing our list, or is that already done in some way?

Reply:
That's already done by the YAML reading library.